Cargo.toml

```diff
@@ -107,7 +107,17 @@ windows = { version = ">=0.54, <=0.57", optional = true }
 libc = "^0.2.164"
 
 [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies]
-core-foundation-sys = "0.8.7"
+objc2-core-foundation = { version = "0.3.0", default-features = false, features = [
+    "std",
+    "CFArray",
```

Comment:
I suppose they're not all used by default? If so, can you enable them like `windows-rs` features only when needed please?

Reply:
Yup, that's done now.